package.json

```diff
@@ -20,7 +20,7 @@
     "seed": "sequelize db:seed:all --seeders-path database/seeders",
     "seed:undo": "sequelize db:seed:undo:all",
     "sequelize": "sequelize",
-    "start": "node dist/bin/www.js",
+    "start": "babel-node index.js",
```

Comment:
#### Critical
- You are adding a level of complexity to this, we shouldn't run `babel-node` in production.

Reply:
Temporarily provided this to ensure that the staging server [apollo-ah-backend-staging.herokuapp.com](https://apollo-ah-backend-staging.herokuapp.com) is in working order, which currently is down. However, I'll resolve changes.